### ISSUE TITLE
Binance credentials wrong format

### ISSUE THREAD

**Issue by marcvsnet** [aleibovici/cryptopump#44]
Using Docker approach this messages appears on providing credentials via the Admin page, which worked before in v1.9.0 and before without issues

```
cryptopump_app    | DEBU[2022-01-06 09:09:34] github.com/aleibovici/cryptopump/exchange.binanceGetAccount - <APIError> code=-2014, msg=API-key format invalid.  orderID=0 threadID=c7ba7esconks7399tk40
cryptopump_app    | DEBU[2022-01-06 09:09:34] github.com/aleibovici/cryptopump/exchange.binanceGetAccount - <APIError> code=-2014, msg=API-key format invalid.  orderID=0 threadID=c7ba7esconks7399tk40
cryptopump_app    | DEBU[2022-01-06 09:09:34] github.com/aleibovici/cryptopump/exchange.binanceGetSymbolFunds - <APIError> code=-2014, msg=API-key format invalid.  orderID=0 threadID=c7ba7esconks7399tk40
cryptopump_app    | DEBU[2022-01-06 09:09:37] github.com/aleibovici/cryptopump/exchange.binanceGetUserStreamServiceListenKey - <APIError> code=-2014, msg=API-key format invalid.  orderID=0 threadID=c7ba7esconks7399tk40
cryptopump_app    | DEBU[2022-01-06 09:09:37] github.com/aleibovici/cryptopump/algorithms.WsUserDataServe - <APIError> code=-2014, msg=API-key format invalid.  orderID=0 threadID=c7ba7esconks7399tk40
```

**Comment by aleibovici** [comment]
@marcvsnet it could be because you are selecting a pair that doesn't exist in Binance TestNet. By default, TestNet is 'true'. Please set it to false and try again.

**Comment by marcvsnet** [comment]
@aleibovici you are right, thanks

**Comment by aleibovici** [comment]
Need implementation of a crypto pair checking function and appropriate error logging to identify when the pair is not available. Will add for next release.

**Comment by aleibovici** [comment]
Implemented in _features_ branch **eed8dec**: Debug log - Balance or Pair not found for symbol. Will make into next Beta and Main.
